lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -72,11 +72,23 @@ private CheckDetail[] createCheckDetailsWithKbvQuality(KBVItem kbvItem)
         var questionState = objectMapper.readValue(kbvItem.getQuestionState(), QuestionState.class);
 
         if (hasPassedWithOneIncorrectAnswer(kbvItem)) {
+            // Scenario:
+            // 1 question was wrong in the first batch
+            // And therefore 2 questions were given in second batch
+            // And they were answered correctly
+            // So we can sort by quality and remove neccessary question
```

Comment:
I think we can you /** */ for multi-line comments some editors allow you to collapse these. 
I think this comment should be in the between line 75 and 76

We create CheckDetails with the lowest 3 kbvQuality values
So we are excluding the highest kbvQuality value

Reply:
Changed the comment and I've added your suggestion - let me know if I've done it correctly